packages/core/src/languages/html/__tests__/css-values.test.ts

```diff
@@ -64,6 +64,27 @@ suite("HTML - CSS Value Expression Factory", function() {
         },
       ],
     },
+    {
+      name: "edge cases",
```

Comment:
Rename this to `"with !important"`

```suggestion
      name: "with !important",
```